src/api/app/controllers/webui/project_controller.rb

```diff
@@ -698,12 +698,16 @@ def add_maintained_project_dialog
 
   def add_maintained_project
     authorize @project, :update?
-    @project.maintained_projects.new(project: @maintained_project)
-    if @project.save
+
+    @maintained_project = params[:maintained_project]
+    maintained_project = Project.where(name: @maintained_project)
```

Comment:
Why not `find_by` instead of` where`? Then you don't need to do `.first` I mean something like:

``` Ruby
maintained_project = Project.find_by(name: params[:maintained_project])
if maintained_project
   @project.maintained_projects.create!(project: maintained_project)
```